OAK-Marketplace-Emotion-Recognition/utils/inference.py

```diff
@@ -0,0 +1,53 @@
+from oak_inference_utils.inference import process_frame
```

Comment:
Please move OAK-Marketplace-Emotion-Recognition/utils/inference.py to OAK-Marketplace-Emotion-Recognition/inference.py